ppr-api/src/ppr_api/reports/v2/report.py

```diff
@@ -184,23 +184,6 @@ def get_large_search_pdf(self):
         report_files['cover.pdf'] = content
         # Merge subreports
         return report_utils.merge_pdfs(report_files), status_code, {'Content-Type': 'application/pdf'}
-        # url = current_app.config.get('REPORT_SVC_URL') + MERGE_URI
-        # headers = {}
-        # token = GoogleStorageTokenService.get_report_api_token()
-        # if token:
-        #    headers['Authorization'] = 'Bearer {}'.format(token)
-        # # response = requests.post(url=url, headers=headers, files=report_files)
-        # req = requests.post(url=url, headers=headers, files=report_files, stream=True)
-        # response = Response(req.iter_content(chunk_size=10*1024), content_type=req.headers['Content-Type'])
-        # current_app.logger.debug(f'Merge search reports response status: {response.status_code}.')
-        # if response.status_code != HTTPStatus.OK:
-        #    # content = ResourceErrorCodes.REPORT_ERR + ': ' + response.content.decode('ascii')
-        #    content = ResourceErrorCodes.REPORT_ERR + ': ' + response.get_data(True)
-        #    current_app.logger.error('Account {0} merge response status: {1} error: {2}.'
-        #                             .format(self._account_id, response.status_code, content))
-        #    return jsonify(message=content), response.status_code, None
-        # # return response.content, response.status_code, {'Content-Type': 'application/pdf'}
-        # return response.get_data(), response.status_code, {'Content-Type': 'application/pdf'}
```

Comment:
For future implementation?

Reply:
Actually it has been replaced and I missed removing it in the previous PR. The subreport pdf's are now merged with a python library instead of making a report api call. GCP has a 32MB limit on response sizes and with the large reports the ppr api to report api call was exceeding this limit when merging reports. Using the python library also slightly improves performance.